3.10/data-science.md

```diff
@@ -0,0 +1,140 @@
+
+Graph Analytics uses an unsupervised
+learning method based on algorithms that are performing analytical processing
+directly on graphs stored in ArangoDB. This 
+[Distributed Iterative Graph Processing (Pregel)](graphs-pregel.html)
+is intended to help you gain analytical insights on
+your data, without having to use external processing systems.
+
+ArangoDB includes the following graph algorithms:
+- [Page Rank](graphs-pregel.html#page-rank): used for ranking documents in a graph
+Search/Traversal
+- [Single-Source Shortest Path](graphs-pregel.html#single-source-shortest-path): calculates
+ the shortest path length between the source and all other vertices.
+ For example, _How to get from a to b_
+- [Hyperlink-Induced Topic Search (HITS)](graphs-pregel.html#hyperlink-induced-topic-search-hits): 
+ a link analysis algorithm that rates Web pages
+- [Vertex Centrality](graphs-pregel.html#vertex-centrality): identifies the most important
+ nodes in a graph. For example, _Who are the influencers in a social network?_
+- [Community Detection](graphs-pregel.html#community-detection): identifies distinct subgroups
+ within a community structure
+
+### GraphML
+
+GraphML capabilities of using more data outperform conventional deep learning
+methods and **solve high-computational complexity graph problems**, such as: 
+- Drug discovery, repurposing, and predicting adverse effects
+- Personalized product/service recommendation
+- Supply chain and logistics
+
+With GraphML, you can also **predict relationships and structures**, such as:
+- Predict molecules for treating diseases (precision medicine)
+- Predict fraudulent behavior, credit risk, purchase of product or services
+- Predict relationships among customers, accounts
+
+ArangoDB is using well-known GraphML frameworks like
+[Deep Graph Library](https://www.dgl.ai){:target="_blank"}
+and [PyTorch Geometric](https://pytorch-geometric.readthedocs.io/en/latest/){:target="_blank"} 
+and connects to these external machine learning libraries. When coupled to
+ArangoDB, you are essentially integrating them with your graph dataset.
+
+## Example: ArangoFlix
+
+ArangoFlix is a complete movie recommendation application that predicts missing
+links between a user and the movies they have not watched yet.
+
+This [interactive tutorial](https://colab.research.google.com/github/arangodb/interactive_tutorials/blob/master/notebooks/Integrate_ArangoDB_with_PyG.ipynb){:target="_blank"} 
+demonstrates how to integrate ArangoDB with PyTorch Geometric to
+build recommendation systems using Graph Neural Networks (GNNs).
+
+The full ArangoFlix demo website is accessible from ArangoDB Oasis,
+the managed cloud for ArangoDB. You can open the demo website that connects to
+your running database from the **Examples** tab of your deployment.
+
+{% hint 'tip' %}
+You can try out ArangoDB Oasis free of charge for 14 days.
+Sign up [here](https://cloud.arangodb.com/){:target="_blank"}. 
```

Comment:
"here" links should be avoided at all costs. It's better to describe what the link points to. Maybe change it to this?
```suggestion
Sign up at [cloud.arangodb.com](https://cloud.arangodb.com/){:target="_blank"}. 
```